app/controllers/admin/creator_settings_controller.rb

```diff
@@ -4,34 +4,33 @@ class CreatorSettingsController < Admin::ApplicationController
                         invite_only_mode logo primary_brand_color_hex public].freeze
 
     def new
+      @creator_settings_form = CreatorSettingsForm.new(
```

Comment:
We send these parameters through as initial values because when we have an error, then we want the values that were set in the form to show up. We also want to pull through the default values for Settings.